Assets/MRTK/SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -170,7 +170,7 @@ public bool EyeCursorSnapToTargetCenter
         /// GameObject eye gaze is currently targeting, updated once per frame.
         /// null if no object with collider is currently being looked at.
         /// </summary>
-        public static GameObject LookedAtTarget { get; private set; }
+        public static GameObject LookedAtTarget => ((CoreServices.InputSystem != null) && (CoreServices.InputSystem.EyeGazeProvider != null)) ? CoreServices.InputSystem.EyeGazeProvider.GazeTarget : null;
```

Comment:
One thing that might make this interesting is that I'm pretty sure the `.GazeTarget` here _might_ actually be from the head gaze (even though it's being queried through `EyeGazeProvider`...). This was previously guarded against in `Update` via

```c#
if ((CoreServices.InputSystem != null) && (CoreServices.InputSystem.EyeGazeProvider != null) &&
    CoreServices.InputSystem.EyeGazeProvider.IsEyeTrackingEnabled &&
    CoreServices.InputSystem.EyeGazeProvider.IsEyeTrackingDataValid)
```

so we might want to add `.IsEyeTrackingEnabled` and `.IsEyeTrackingDataValid` checks here too.